### ISSUE TITLE
Accessibility Report: KSS Generated Site

### ISSUE THREAD

**Issue by sherakama** [SU-SWS/decanter#327]
Use the AMP tool to scan the entire decanter.stanford.edu for accessibility issues and document them to this ticket. Although not our code, we can and should fix any accessibility issues that arise from the scan.

**Comment by cjwest** [comment]
We have an automated test at:

**Comment by cjwest** [comment]
Because I'm finding the navigation within AMP rather confusing, I created a map for Tests:
![Screen Shot 2019-03-15 at 4 29 58 PM](https://user-images.githubusercontent.com/284440/54467000-b95e7d80-473f-11e9-8085-e1c8f955882d.png)

**Comment by cjwest** [comment]
We met with folks from Level Access. They indicated that further testing should be done manually.
We are looking into training on using the manual portion of Level Access.

**Comment by cjwest** [comment]
@sherakama, Clare O'Keeffe has completed manual testing.

> Hi Jiatyan,

> Manual testing of the Decanter library components is complete but for two items. I did not see working examples of screen reader only text and screen reader only elements that I could test. If you can make those available I will test them. Everything else is ready for review at: https://stanford.levelaccess.net/public/reporting/view_report.php?report_id=283478. I believe we are scheduled to go over the results on the 18th during your usual call with Kevin and Rachael.

> Please let me know if you have any questions.

> Thanks,

> Clare O'Keeffe, CPWA
> Manager, Accessibility Consulting
> Level Access
> clare.okeeffe@levelaccess.com

**Comment by cjwest** [comment]
@sherakama ,

Would you like me to create issues for each of these?
https://stanford.levelaccess.net/public/reporting/view_instances.php?report_id=283478

**Comment by sherakama** [comment]
OOoo yay! 

Thanks for this. Tickets! Yes, Tickets!

**Comment by sherakama** [comment]
This is no longer relevant.
